restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -1031,4 +1036,126 @@ public static String maybeQuote(String text) {
       return text;
     return '"' + PATTERN_DOUBLE_QUOTE.matcher(text).replaceAll(ESCAPED_DOUBLE_QUOTE) + '"';
   }
+
+  public static Map<String, String> convertColumnMap(UserDefinedType udt) {
+    return udt.columnMap().entrySet().stream()
```

Comment:
Is this safe or do we need a null check on `udt` beforehand?

Reply:
Good catch.